docker-compose.yml

```diff
@@ -1,39 +1,90 @@
 volumes:
-  postgres_data:
+  postgres-data:
     driver: local
-  redis_data:
+  redis-data:
     driver: local
+
 services:
+  rating-system-service:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: rating-system-service:latest
+    container_name: rating-system-service
+    restart: unless-stopped
+    environment:
+      - SPRING_PROFILES_ACTIVE=local
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:5432/postgres
+      - SPRING_DATASOURCE_USERNAME=admin
+      - SPRING_DATASOURCE_PASSWORD=9tRQF5H2TsNM
+      - SPRING_DATA_REDIS_HOST=redis
+      - SPRING_DATA_REDIS_PORT=6379
+      - SPRING_DATA_REDIS_PASSWORD=your_secure_password
+      - SERVER_PORT=9090
+      - KEYCLOAK_REALM=leverx
+      - KEYCLOAK_AUTH_SERVER_URL=http://keycloak:8080
+      - KEYCLOAK_RESOURCE=rating-system
+      - MASTER_USER_USERNAME=admin
+      - MASTER_USER_PASSWORD=admin
+      - MASTER_REALM=master
+      - MASTER_CLIENT=admin-cli
+      - MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE=health,info,metrics
+    networks:
+      - leverx-network
+    depends_on:
+      - redis
+      - postgres
+      - keycloak
+
   postgres:
-    image: postgres
-    container_name: rating-system-pg
-    volumes:
-      - postgres_data:/var/lib/postgresql/data
+    image: postgres:latest
+    container_name: postgres
     environment:
-      POSTGRES_DB: keycloak
-      POSTGRES_USER: keycloak
-      POSTGRES_PASSWORD: password
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: 9tRQF5H2TsNM
```

Comment:
We don't store user and password in files. Please provide them during the execution of docker-compose up